plugins/generator-1.19.4/forge-1.19.4/templates/elementinits/villagerprofessions.java.ftl

```diff
@@ -50,7 +50,7 @@ public class ${JavaModName}VillagerProfessions {
 			registerProfession(
 					"${villagerprofession.getModElement().getRegistryName()}",
 					${mappedBlockToBlock(villagerprofession.pointOfInterest)},
-					new SoundEvent(new ResourceLocation("${villagerprofession.actionSound}"), new ResourceLocation("${villagerprofession.actionSound}"), true)
+					new SoundEvent(new ResourceLocation("${villagerprofession.actionSound}"), 16.0F, true)
```

Comment:
There is a more suitable method for this, check how sounds are handled in 1.19.4 generator

Reply:
Ok, it's done